enterprise_access/apps/bffs/tests/test_handlers.py

```diff
@@ -0,0 +1,125 @@
+from django.test import RequestFactory, TestCase
+from faker import Faker
+from rest_framework.exceptions import ValidationError
+
+from enterprise_access.apps.bffs.context import HandlerContext
+from enterprise_access.apps.bffs.handlers import BaseHandler, BaseLearnerPortalHandler, DashboardHandler
+from enterprise_access.apps.core.tests.factories import UserFactory
+
+
+class TestBaseHandler(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.factory = RequestFactory()
+        self.mock_user = UserFactory()
+        self.faker = Faker()
+
+        self.mock_enterprise_customer_uuid = self.faker.uuid4()
+        self.request = self.factory.get('sample/api/call')
+        self.request.query_params = {
+            'enterprise_customer_uuid': self.mock_enterprise_customer_uuid
+        }
+        self.request.user = self.mock_user
+        self.context = HandlerContext(self.request)
```

Comment:
Given these lines seem to be repeated fairly verbatim throughout the different test classes, I wonder if it'd make sense to create something like a `TestBaseHandlerMixin` that test cases to extend to be a bit more DRY?

```python
class TestBaseHandlerMixin(TestCase):
    def setUp(self):
        super().setUp()
        self.factory = RequestFactory()
        self.mock_user = UserFactory()
        self.faker = Faker()

        self.mock_enterprise_customer_uuid = self.faker.uuid4()
        self.request = self.factory.get('sample/api/call')
        self.request.query_params = {
            'enterprise_customer_uuid': self.mock_enterprise_customer_uuid
        }
        self.request.user = self.mock_user
        self.context = HandlerContext(self.request)

class TestBaseHandler(TestBaseHandlerMixin):
    # ...
```